msal/src/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -39,28 +39,33 @@ public class AuthenticationResult
     {
         private const string Oauth2AuthorizationHeader = "Bearer ";
         private readonly MsalAccessTokenCacheItem _msalAccessTokenCacheItem;
+        private readonly MsalIdTokenCacheItem _msalIdTokenCacheItem;
```

Comment:
>_msalIdTokenCacheIte [](start = 46, length = 20)

as far as I can see in most other places we do not use _ / can we pls. stick to using either or? (personal preferences is without the _)